orm/index_test.go

```diff
@@ -164,7 +164,7 @@ func TestCounterMultiKeyIndex(t *testing.T) {
 			expKeys:  [][]byte{encodeSequence(6), []byte("even")},
 			expError: false,
 		},
-		{ // id miss-match
+		{ // id missmatch
```

Comment:
it's mismatch with one s. otherwise good

Reply:
Silly me. Editing straight from Github!